tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -46,27 +50,43 @@ def parameters(cls):
                 value_type=int, min_value=1, optional=True, default=1,
                 description="The number of classes with the highest probability, which will be used to decide "
                             "if prediction is correct."
-            )
+            ),
+            'match': BoolField(optional=True, default=False)
         })
 
         return parameters
 
     def configure(self):
         self.top_k = self.get_value_from_config('top_k')
+        self.match = self.get_value_from_config('match')
```

Comment:
please add check that accuracy_score is not None in case when match=False

Reply:
fixed